src/containers/Operations/columns.tsx

```diff
@@ -114,5 +126,77 @@ export function getColumns(): DataTableColumn<TOperation>[] {
                 return Date.now() - createTime;
             },
         },
+        {
+            name: 'Actions',
+            sortable: false,
+            resizeable: false,
+            header: '',
+            render: ({row}) => {
+                return (
+                    <OperationsActions
+                        operation={row}
+                        database={database}
+                        refreshTable={refreshTable}
+                    />
+                );
+            },
+        },
     ];
 }
+
+interface OperationsActionsProps {
+    operation: TOperation;
+    database: string;
+    refreshTable: VoidFunction;
+}
+
+function OperationsActions({operation, database, refreshTable}: OperationsActionsProps) {
+    const [cancelOperation, {isLoading: isLoadingCancel}] =
```

Comment:
As for me it would be nice to show success toast if action is completed. What do you think?